test/e2e/utils/test_context.go

```diff
@@ -56,12 +56,11 @@ type TestContext struct {
 
 // NewTestContext init with a random suffix for test TestContext stuff,
 // to avoid conflict when running tests synchronously.
-func NewTestContext(binaryName string, env ...string) (*TestContext, error) {
+func NewTestContext(binaryName, domain, dir string, env ...string) (*TestContext, error) {
```

Comment:
Same here 
Why would we need to make this change?
I think we can revert this one, right?